optimizely/config_manager.py

```diff
@@ -246,8 +246,8 @@ def set_update_interval(self, update_interval):
                 'Invalid update_interval "{}" provided.'.format(update_interval)
             )
 
-        # If polling interval is less than minimum allowed interval then set it to default update interval.
-        if update_interval < enums.ConfigManager.MIN_UPDATE_INTERVAL:
+        # If polling interval is less than or equal to 0 then set it to default update interval.
+        if update_interval <= 0:
             self.logger.debug('update_interval value {} too small. Defaulting to {}'.format(
```

Comment:
`update_interval value is not greater than zero, Defaulting to {} value`. @aliabbasrizvi WDYT?

Reply:
I think current message is fine.